CONTRIBUTING.md

```diff
@@ -1,43 +1,43 @@
-# Contribuer à la librairie everythingToEverywhere AXA France
+# Contributing to the everythingToEverywhere library AXA France
 
-Merci de considérer la contribution au repo everythingToEverywhere ! Nous sommes ravis de voir que vous souhaitez participer à l'amélioration de ce projet.
+Thank you for considering contributing to the everythingToEverywhere repo! We are excited to see that you want to participate in improving this project.
 
-## Démarrage du projet
+## Getting Started
 
-Le build du projet se fait via la commande `npm run build`. Pour tester le package en local, vous pouvez faire une installation locale grâce à la commande `npm i <local_path>`.
+The project build is done via the command `npm run build`. To test the package locally, you can perform a local installation using the command `npm i <local_path>`.
 
-## Comment contribuer
+## How to Contribute
 
-1. **Fork du Projet :** Commencez par forker le projet vers votre compte GitHub.
+1. **Fork the Project:** Start by forking the project to your GitHub account.
 
-2. **Cloner le Projet :** Clonez le projet depuis votre compte vers votre machine locale.
+2. **Clone the Project:** Clone the project from your account to your local machine.
 
 ```bash
-git clone https://github.com/votre-utilisateur/everythingToEverywhere.git
+git clone https://github.com/your-username/everythingToEverywhere.git
 ```
 
-3. **Créer une Branche :** Créez une branche pour travailler sur votre nouvelle fonctionnalité ou correction.
+3. **Create a Branch:** Create a branch to work on your new feature or fix.
 
 ```bash
-git switch -c nom-de-ma-branche
+git switch -c my-branch-name
 ```
 
-1. **Faire des Modifications :** Effectuez les modifications nécessaires et assurez-vous que votre code respecte les conventions du projet. Utilisez la [Conventional Commits](https://www.conventionalcommits.org/) pour le nommage et la découpe des commits.
+4. **Make Changes:** Make the necessary changes and ensure your code adheres to the project's conventions. Use [Conventional Commits](https://www.conventionalcommits.org/) for naming and splitting commits.
```

Comment:
Suggestion: "and ensure your code **follows the** project's conventions."

Reply:
thanks ! done